observation/protocol/morse.pb.go

```diff
@@ -31,8 +31,10 @@ const (
 	MorseEndpointErrorType_MORSE_ENDPOINT_ERROR_MISCONFIGURED                       MorseEndpointErrorType = 4
 	MorseEndpointErrorType_MORSE_ENDPOINT_ERROR_TLS_CERTIFICATE_VERIFICATION_FAILED MorseEndpointErrorType = 5
 	MorseEndpointErrorType_MORSE_ENDPOINT_ERROR_INVALID_RESPONSE                    MorseEndpointErrorType = 6
-	MorseEndpointErrorType_MORSE_ENDPOINT_ERROR_NON_JSON_RESPONSE                   MorseEndpointErrorType = 7 // Response returned from endpoint is not valid JSON
-	MorseEndpointErrorType_MORSE_ENDPOINT_ERROR_INTERNAL                            MorseEndpointErrorType = 8 // Added for internal gateway errors
+	// This error indicates that the endpoint returned invalid data in its response and was rejected by the SDK.
```

Comment:
These should be added to the .proto file, or they will be overwritten each time we regenerate `.pb.go` files.

Reply:
I've confirmed that it is added to the .proto file.